packages/flutter_tools/test/flutter_manifest_test.dart

```diff
@@ -478,6 +481,25 @@ flutter:
         expectedBuildNumber: null,
       );
     });
+
+    testUsingContext('returns proper error when font detail is malformed', () async {
```

Comment:
Might add more context or a link to the original issue in a comment, since the error case is hard to parse out visually

Reply:
Great point. Thanks, Done.